Index: elmerfem/eio/Makefile.am
===================================================================
--- elmerfem.orig/eio/Makefile.am	2010-03-07 10:23:24.000000000 -0500
+++ elmerfem/eio/Makefile.am	2010-03-07 10:25:54.000000000 -0500
@@ -1,2 +1,4 @@
 SUBDIRS=include src
 EXTRA_DIST=GPL-2
+
+DISTCLEANFILES = aclocal.m4
Index: elmerfem/elmerparam/Makefile.am
===================================================================
--- elmerfem.orig/elmerparam/Makefile.am	2010-03-07 10:23:24.000000000 -0500
+++ elmerfem/elmerparam/Makefile.am	2010-03-07 10:25:54.000000000 -0500
@@ -5,3 +5,5 @@
 dist-hook:
 	cd $(distdir)/doc; \
 	make clean
+
+DISTCLEANFILES = config.log config.status
Index: elmerfem/fem/Makefile.am
===================================================================
--- elmerfem.orig/fem/Makefile.am	2010-03-07 10:23:24.000000000 -0500
+++ elmerfem/fem/Makefile.am	2010-03-07 10:25:54.000000000 -0500
@@ -17,6 +17,4 @@
 #	  sh nodist.sh;                        \
 #	fi
 
-
-
-
+CLEANFILES = a.out
Index: elmerfem/fem/src/Makefile.am
===================================================================
--- elmerfem.orig/fem/src/Makefile.am	2010-03-07 10:25:54.000000000 -0500
+++ elmerfem/fem/src/Makefile.am	2010-03-07 10:25:54.000000000 -0500
@@ -214,7 +214,7 @@
 endif
 
 clean-local:
-	$(RM) $(ALL) *.o *.mod *.f90 *.F *.fpp *~ *.a *.so *.dll \#* mpif mpif.h ElmerSolver ElmerSolver_mpi
+	$(RM) $(ALL) *.o *.mod *.f90 *.F *.fpp *~ *.a *.so *.dll \#* mpif mpif.h ElmerSolver ElmerSolver_mpi GebhardtFactors ResultToResult ViewFactors
 
 
 if HAVE_WINDRES
Index: elmerfem/fem/src/binio/Makefile.am
===================================================================
--- elmerfem.orig/fem/src/binio/Makefile.am	2010-03-07 10:23:24.000000000 -0500
+++ elmerfem/fem/src/binio/Makefile.am	2010-03-07 10:25:54.000000000 -0500
@@ -12,3 +12,5 @@
 install-exec-local:
 	mkdir -p $(DESTDIR)$(ELMER_SOLVER_DATADIR)/include
 	$(INSTALL) *.mod $(DESTDIR)$(ELMER_SOLVER_DATADIR)/include
+
+CLEANFILES = *.mod
Index: elmerfem/hutiter/Makefile.am
===================================================================
--- elmerfem.orig/hutiter/Makefile.am	2010-03-07 10:23:24.000000000 -0500
+++ elmerfem/hutiter/Makefile.am	2010-03-07 10:25:54.000000000 -0500
@@ -1 +1,3 @@
 SUBDIRS = src include examples
+
+DISTCLEANFILES = aclocal.m4
Index: elmerfem/elmerparam/src/matlab/Makefile.in
===================================================================
--- elmerfem.orig/elmerparam/src/matlab/Makefile.in	2010-03-07 10:23:24.000000000 -0500
+++ elmerfem/elmerparam/src/matlab/Makefile.in	2010-03-07 10:25:54.000000000 -0500
@@ -12,4 +12,6 @@
 clean:
 	rm -f elmer_param.@mex_ext@
 
+maintainer-clean: clean
+
 check:
Index: elmerfem/elmerparam/src/R/Makefile.in
===================================================================
--- elmerfem.orig/elmerparam/src/R/Makefile.in	2010-03-07 10:25:54.000000000 -0500
+++ elmerfem/elmerparam/src/R/Makefile.in	2010-03-07 10:29:01.000000000 -0500
@@ -9,6 +9,8 @@
 	cp -r ./* $(distdir)/
 
 clean:
-	rm -f elmerparam/src/*.{o,so}
+	rm -f elmerparam/src/*.o elmerparam/src/*.so *.tar.gz
+
+maintainer-clean: clean
 
 check:
Index: elmerfem/elmerparam/src/Makefile.am
===================================================================
--- elmerfem.orig/elmerparam/src/Makefile.am	2010-03-07 10:30:50.000000000 -0500
+++ elmerfem/elmerparam/src/Makefile.am	2010-03-07 10:31:17.000000000 -0500
@@ -37,3 +37,5 @@
 	$(INSTALL) elmerparam.mod $(DESTDIR)$(includedir)
 endif
 MOSTLYCLEANFILES = $(nodist_include_HEADERS)
+
+CLEANFILES = *.mod
